tools/GeneratePythonLibrary.sh

```diff
@@ -110,7 +110,7 @@ function create-local-creds {
 }
 
 function generate-from-swagger {
-  java -jar ./tools/swagger-codegen-cli.jar generate -i $SWAGGER_YAML -c $SWAGGER_CONFIG -l python || true
+#  java -jar ./tools/swagger-codegen-cli.jar generate -i $SWAGGER_YAML -c $SWAGGER_CONFIG -l python || true
```

Comment:
Might be nice to clean this up, both removing commented out code and rename `generate-from-swagger` if that's not what happening.

Reply:
Done, but I have additional work to do here.